plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java

```diff
@@ -223,20 +225,20 @@ public void configure() throws ConfiguratorException {
 
     private List<String> getStandardConfig() {
```

Comment:
the whole logic to retrieve yaml sources should be moved to getConfigurationPath as we now have a component to manage it